content/en/_index.md

```diff
@@ -218,6 +310,18 @@ a file or a specific action is not.
         // Snippet content ...
     }
   {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    function main(requiredArg) {
+      // This argument is required, and should be declared
+      const requiredArg = '...';
+
+      exampleSnippet(requiredArg)
+    }
+
+    const exampleSnippet = (requiredArg) => {
+      // Snippet content...
+    }
```

Comment:
It also seems like this should be consistent with the Java sample, and just show the `exampleSnippet` bit

Reply:
Node methods don't have explicit return types, so I'd argue we should _at least_ call out that we're not returning anything here.

(In Java, that's implied by `void`.)

Reply:
Does calling the function without assignment imply that it's not returning a value or just that it's not being used? This isn't typically the case for most languages I'm familiar with. 

or were you thinking something like a comment in `exampleSnippet`? Maybe like `// Avoid using "return"... instead rely on testing output from "console.log("")"`

Reply:
The latter - I added this as a comment:
> // (This snippet should not `return` anything.)